install_config/adding_hosts_to_existing_cluster.adoc

```diff
@@ -309,3 +309,152 @@ $ ansible-playbook [-i /path/to/file] \
 
 . After the playbook completes successfully,
 xref:../install_config/install/advanced_install.adoc#advanced-verifying-the-installation[verify the installation].
+
+[[replacing-existing-masters]]
+== Replacing Existing Masters with etcd Colocated
+
+Follow these steps when you are migrating your machines to a different data
+center and the network and IPs assigned to it will change.
+
+. Back up the primary xref:../admin_guide/backup_restore.adoc#etcd-backup[etcd]
+and xref:../admin_guide/backup_restore.adoc#master-backup[master] nodes.
++
+[IMPORTANT]
+====
+Ensure that you back up the *_/etc/etcd/ca/_* directory, as noted in the
+xref:../admin_guide/backup_restore.adoc#backup-restore-adding-etcd-hosts[Backup
+and Restore] topic.
+====
+
+. Provision as many new machines as there are masters to replace.
+
+. Add or expand the cluster. for example, if you want to add 3 masters with etcd
+colocated, scale up 3 master nodes or 3 etcd nodes.
+
+.. Add a xref:adding-nodes-advanced[master]. In step 3 of that process, add the
+host of the new data center in `[new_masters]` and `[new_nodes]` and run the
+xref:adding-nodes-advanced[master *_scaleup.yml_* playbook].
+
+.. Put the same host in the xref:adding-etcd-hosts-to-existing-cluster[etcd]
+section and run the etcd *_scaleup.yml_* playbook.
+
+.. Verify that the host was added:
++
+----
+# oc get nodes
+----
+
+.. Verify that the master host IP was added:
++
+----
+# oc get ep kubernetes
+----
+
+.. Verify that etcd was added. The value of `ETCDCTL_API` depends on the version
+being used:
++
+----
+# ETCDCTL_API=2
+# source /etc/etcd/etcd.conf
+# etcdctl --cert-file=$ETCD_PEER_CERT_FILE --key-file=$ETCD_PEER_KEY_FILE \
```

Comment:
Move the ETCDCTL_API=2 to the same line as the etcdctl command so that it doesn't persist. example
`# ETCDCTL_API=2 etcdctl --cert-file=$ETCD_PEER_CERT_FILE --key-file=$ETCD_PEER_KEY_FILE \`
...